doc-src/install2.rst

```diff
@@ -0,0 +1,124 @@
+   following tables could be a section with a bullet list.
+
+   A third alternative is, don't document this here, but have
+   :code:`configure` provide a report saying what you can and can't do on a
+   given system.
+
+.. code-block:: none
+
+                                               POSIX environment
+                                               |  C11 compiler
+                                               |  |  Git
+                                               |  |  |  GNU Autotools
+                                               |  |  |  |  Sphinx 1.4.9+
+                                               |  |  |  |  |  Python 3.4+
+   BUILDING CHARLIECLOUD ..................... |  |  |  |  |  |
+   build Charliecloud from source              x  x
+   bootstrap build from Git clone              x  x  x  x
+   re-build documentation [1]                  x           x  x
+   build test suite                            x              x
+
+                                               POSIX environment
+                                               |  Bash 4.1+
+                                               |  |  Docker
+                                               |  |  |  mktemp(1)
+                                               |  |  |  |  Buildah 1.10.1+
+                                               |  |  |  |  |  Python 3.4+
+                                               |  |  |  |  |  |  Python module "lark-parser"
+                                               |  |  |  |  |  |  |  skopeo
+                                               |  |  |  |  |  |  |  |  umoci
+   IMAGE BUILDERS ............................ |  |  |  |  |  |  |  |  |
+   Docker                                      x  x  x  x
+   Buildah                                     x  x        x
+   ch-grow                                     x  x           x  x  x  x
+
+                                               POSIX environment
+                                               |  Bash 4.1+
+                                               |  |  One of the image builders above
+                                               |  |  |  Access to image repository
+                                               |  |  |  |  SquashFS tools
+   MANAGING CONTAINER IMAGES ................  |  |  |  |  |
+   build images from Dockerfile with ch-build  x  x  x  x
+   push/pull images to/from builder storage    x  x  x  x
+   pack image with ch-builder2tar              x  x  x
+   pack image with ch-builder2squash           x  x  x     x
+
+                                               POSIX environment
+                                               |  user namespaces
+                                               |  |  SquashFUSE
+   RUNNING CONTAINERS .......................  |  |  |
+   ch-run                                      x  x
+   unpack image tarballs                       x
+   mount/unmount SquashFS images               x     x
```

Comment:
I think the `unpack image tarballs` and `mount/unmount SquashFS images` should go in `MANAGING CONTAINER IMAGES`.